.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## So you've found a problem in CatBlock?
+### Answer these questions to help us solve the case!
+
+### 1. List some steps we can follow to make the bug happen on our end.
+
+1. 
+2. 
+3. 
+
+### 2. When you followed these steps above, what did you think should happen?
+
+
+
+### 3. What actually happened instead?
+
+
+### 4. Finally, click the CatBlock button, then Options, then Support on the page that appears. Provide us with the **debug info** from that page.
```

Comment:
@kpeckett "Debug info" is not specific enough, include some additional text, how to obtain debug info from within the Support tab e.g. "click on the debug info button".